cove/lib/common.py

```diff
@@ -234,3 +234,98 @@ def get_schema_validation_errors(json_data, schema_url, schema_name, current_app
         unique_validator_key = [validator_type, message, path_no_number]
         validation_errors[json.dumps(unique_validator_key)].append(value)
     return dict(validation_errors)
+
+
+def _get_schema_deprecated_paths(schema_name, schema_url, obj=None, current_path=(), deprecated_paths=None):
+    '''Get a list of deprecated paths (as tuples) in a schema'''
+    if deprecated_paths is None:
+        deprecated_paths = []
+
+    if schema_url and schema_url.startswith("http"):
+        obj = jsonref.loads(requests.get(schema_url + schema_name).text)
```

Comment:
I think we want to be using CustomJsonrefLoader here https://github.com/OpenDataServices/cove/blob/043f8e9cfeed32bd4c093d3080997a5bedb6e6d0/cove/lib/common.py#L101 (the class I link to should probably have a docstring explaining why!)